From 875690e5d7837568dd05bda3bffd9716adcd9c2c Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@z3ntu.xyz>
Date: Fri, 20 Mar 2020 21:58:56 +0100
Subject: [PATCH 3/3] Fix CMake error with Qt 5.14

CMake Error at CMakeLists.txt:136 (_populate_Gui_plugin_properties):
  _populate_Gui_plugin_properties Macro invoked with incorrect arguments for
  macro named: _populate_Gui_plugin_properties
---
 CMakeLists.txt | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 189b6f03..89b4c249 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -140,7 +140,11 @@ include(GNUInstallDirs)
 # Determine QPA plugin install path
 # FIXME(greyback) there must be a better way than calling a private macro to set a property.
 # I'm manually calling a macro from Qt5GuiConfig.cmake, but I don't understand why it isn't being called.
-_populate_Gui_plugin_properties("Gui" "PLATFORMS" "platforms")
+if (Qt5Gui_VERSION VERSION_LESS 5.14.0)
+  _populate_Gui_plugin_properties("Gui" "PLATFORMS" "platforms")
+else()
+  _populate_Gui_plugin_properties("Gui" "PLATFORMS" "platforms" FALSE)
+endif()
 get_target_property(Qt5Gui_QPA_Plugin_Path Qt5::Gui IMPORTED_LOCATION_PLATFORMS)
 
 # Set QML module install path
-- 
2.25.2

